### PR TITLE
Fix queries and constants organization in Noobaa Plugin

### PR DESCRIPTION
**Pull request by bipuladh**

### CODE REVIEW

File: frontend/packages/noobaa-storage-plugin/src/constants/strings.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export const BUCKET_CLASS = 'Bucket Class';
+export const OCS_NS = 'openshift-storage';
+export const NB_PROVISIONER = 'noobaa.io/obc';
+
+export const CHART_LABELS = {
+  [BY_LOGICAL_USAGE]: 'Logical used capacity per account',
+  [BY_PHYSICAL_VS_LOGICAL_USAGE]: 'Physical vs. Logical used capacity',
+  [BY_EGRESS]: 'Egress Per Provider',
+  [BY_IOPS]: 'I/O Operations count',
+};
```
Comment (afreen23) [inline frontend/packages/noobaa-storage-plugin/src/constants/strings.ts:17]: This should be in a utils folder for data consumption card

File: frontend/packages/noobaa-storage-plugin/src/queries.ts [hunk 2]
```diff
@@ -2,21 +2,26 @@ import { ProjectModel } from '@console/internal/models';
 import { PROJECTS, BUCKET_CLASS } from './constants';
```
Comment (cloudbehl) [inline frontend/packages/noobaa-storage-plugin/src/queries.ts:2]: can we call it `dashboard-queries.ts` to be more clear?

Comment (afreen23) [reply frontend/packages/noobaa-storage-plugin/src/queries.ts:2]: Its possible that other noobaa components would use same queries in future, so we can keep it as it is for now.

File: frontend/packages/noobaa-storage-plugin/src/constants/data-consumption.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+export const ACCOUNTS = 'Accounts';
+export const PROVIDERS = 'Providers';
+export const BY_IOPS = 'I/O Operations';
+export const BY_LOGICAL_USAGE = 'Logical Used Capacity';
+export const BY_PHYSICAL_VS_LOGICAL_USAGE = 'Physical Vs Logical Usage';
+export const BY_EGRESS = 'Egress';
+export const PROJECTS = 'Projects';
+export const BUCKET_CLASS = 'Bucket Class';
+export const OCS_NS = 'openshift-storage';
+export const NB_PROVISIONER = 'noobaa.io/obc';
```
Comment (afreen23) [inline frontend/packages/noobaa-storage-plugin/src/constants/data-consumption.ts:10]: This should be not be in data-consumption, rather in a generic file e.g `common.ts`

File: frontend/packages/noobaa-storage-plugin/src/constants/data-consumption.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+export const ACCOUNTS = 'Accounts';
+export const PROVIDERS = 'Providers';
+export const BY_IOPS = 'I/O Operations';
+export const BY_LOGICAL_USAGE = 'Logical Used Capacity';
+export const BY_PHYSICAL_VS_LOGICAL_USAGE = 'Physical Vs Logical Usage';
+export const BY_EGRESS = 'Egress';
+export const PROJECTS = 'Projects';
+export const BUCKET_CLASS = 'Bucket Class';
```
Comment (afreen23) [inline frontend/packages/noobaa-storage-plugin/src/constants/data-consumption.ts:8]: These two we can later move to breakdown file.

**Pull request merged**